boards/arm/stm32/b-g474e-dpow1/src/Make.defs

```diff
@@ -0,0 +1,38 @@
+############################################################################
+# boards/arm/stm32/b-g474e-dpow1/src/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+-include $(TOPDIR)/Make.defs
```

Comment:
```suggestion
include $(TOPDIR)/Make.defs
```
To make it consistent with the recent changes to other makefiles. (#1088)

Reply:
Good catch! Thank you reviewing and noticing that. I'll push a commit that fixes it soon...